File: Taller de Haskell/Haskell/problem3.hs
division :: Double -> Double -> String
division _ 0 = "Error: No se puede dividir por cero."
division a b = "Resultado: " ++ show (a / b)

main :: IO ()
main = do
    putStrLn "Ingrese el dividendo: "
    aStr <- getLine
    putStrLn "Ingrese el divisor: "
    bStr <- getLine
    let a = read aStr :: Double
    let b = read bStr :: Double
    putStrLn (division a b)

File: Taller de Haskell/Python/problem3.py
# Calculadora de división

# Bloque try-except
 
try:
    a = float(input("Ingrese el dividendo: "))
    b = float(input("Ingrese el divisor: "))

    resultado = a / b
    print("El resultado de la división es:", resultado)

# Mensaje de error 
except ZeroDivisionError:
    print("Error: No se puede dividir por cero.")